### PR TITLE
header and navbar

### PR DESCRIPTION
**Pull request by aakbarkhan**
Added the headline section and the navbar

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @aakbarkhan , 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+</head>
+<body>
+
+    <header class="heading">
+        <a class="logo" href="#"> 
+            John Doe
+        </a>
+        <a href="#">
+            <img class="menu" src="./assets/menu_white_24dp.svg" alt="menu_bar">
+        </a>
```
Comment (mke2111) [inline index.html:18]: - [ ] For the hamburg icon, enclose it within `nav` tags. Remember it still has to be clickable

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <div class="person-name">
+            <h1> Aku</h1>
+            <h2>Software Developer</h2>
+        </div>
+        <img class="ellipse" src="./assets/Ellipse18.png" alt="">
+
+    </div>
+    <p class="discription">
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hasitate to contact me.
+    </p>
```
Comment (mke2111) [inline index.html:30]: - [ ] Add the description class paragraph inside to the div class of job-title.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    </div>
+    <hr>
+    <div class="social-icons">
+        <img src="./assets/git.png" alt="">
+        <img src="./assets/ln.png" alt="">
+        <img src="./assets/v.png" alt="">
+        <img src="./assets/tw.png" alt="">
+        <img src="./assets/m.png" alt="">
+    </div>
```
Comment (mke2111) [inline index.html:71]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
+body {
+  height: auto;
+  background-color: #210b54;
+}
```
Comment (mke2111) [inline style.css:4]: - [ ] For these sections, focus  on the the margins (top, bottom, left, right) defined in the design
- [ ] Your page
Screenshot from 2021-09-17 12-24-14

Original Page.
Screenshot from 2021-09-17 12-25-04

File: index.html (hunk 1 above)
Comment (aakbarkhan) [reply index.html]: I have made the changes. Thank you so much.

File: index.html (hunk 2 above)
Comment (aakbarkhan) [reply index.html:32]: Thank you.

File: style.css (hunk 4 above)
Comment (aakbarkhan) [reply style.css]: Thank you.

File: index.html (hunk 3 above)
Comment (aakbarkhan) [reply index.html]: Done

File: README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: mobile version
+
+> This project is Portfolio mobile version.
+
```
Comment (george-shammar) [inline README.md:6]: [OPTIONAL]

- [ ] Please, always commit your work in the present tense. All your commit messages have been done in the past tense. Kindly take note of this. :pray:

**Review by george-shammar — APPROVED** [review]
Hi @aakbarkhan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: Thank you for making the required changes. :+1: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (aakbarkhan) [reply README.md:6]: Thank you.

**Pull request merged**